### PR TITLE
Fix BrowserStack / Safari timeout.

### LINKED ISSUE #185: Test "does not raise element-conflict error" unstable on React tester CI on Safari
## Are you reporting a feature request or a bug?

Unstable test

## Details

"does not raise element-conflict error" test turned out to be unstable during the development of #166. It was failing randomly on various React and Node versions. We should probably try to investigate it if it turns out to be notorious.

### PR DESCRIPTION
**Pull request by MMMalik**
Let me enumerate a couple of initial observations:

1. The issue is observed for BrowserStack browsers (Safari and Edge).
2. The timeout issue occurs for the first test case that involves editor loading, i.e. `does not raise element-conflict error`. 
3. There seems to be no specific React version associated with this bug.

I did some debugging locally with BrowserStack and I noticed that on Safari (and sometimes on Edge too), the test case `does not raise element-conflict error` got stuck inside `waitForEditor`'s `tick` function. Time measurements revealed that it took several seconds before `editor` instance is available for the component and that was the bottleneck.

It looks like the process of loading and processing an external script via a BrowserStack browser is quite slow. I'm not sure if we can improve that somehow or that it is just slow due to BrowserStack's infrastructure, firewalls, etc.

For now I'm proposing two small adjustments to `karma.conf.js`:

1. Load ckeditor script as dependency, so that a browser on BrowserStack can at least cache that script for subsequent tests.
2. Increase `timeout` to provide a safer margin.

Closes #185.

### CODE REVIEW

**Comment by MMMalik** [comment]
I noticed that a test suite might randomly fail for yet another reason, such as connectivity issues with BrowserStack. Re-running failed job(s) helps.

One way to test my changes is to open Travis CI and manually re-run jobs for this PR. I did that several times and I haven't observed any issues.

**Review by f1ames — CHANGES_REQUESTED** [review]
On the first CI job re-run I got (see [here](https://travis-ci.com/github/ckeditor/ckeditor4-react/jobs/496947719#L271)):

image

But the error says about 2000ms timeout :thinking: Maybe there is another timeout related to Karma waiting to connect to the browser or to BrowserStack connection itself?

**Comment by MMMalik** [comment]
That is yet another connectivity issue: browser gets connected but disconnects afterwards. I focused on the issue that was causing `does not raise element-conflict error` test case to fail.

Since connectivity issues are to be handled in #191 (either by dropping BS or fixing its problems), we can put this on hold for now.

**Comment by f1ames** [comment]
> That is yet another connectivity issue: browser gets connected but disconnects afterwards. I focused on the issue that was causing `does not raise element-conflict error` test case to fail.
> 
> Since connectivity issues are to be handled in #191 (either by dropping BS or fixing its problems), we can put this on hold for now.

I've restarted the CI once again. I guess if it's green we can merge it already - one random issue less and the fixes applied here will be still valid with #191. So it makes sense to merge it IMHO.

**Comment by f1ames** [comment]
Rebased onto latest `master`.

**Review by f1ames — APPROVED** [review]
CI is green (well, it was before the rebase), so lets merge it :+1: Good job :+1:

**Pull request merged**
